### PR TITLE
🎬 add sync action

### PR DESCRIPTION
**Pull request by dmiller15**
## Description

Adds github action to sync pull request template

Part of https://github.com/d3b-center/bixu-tracker/issues/727

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

Needs to be merged to master to test.
Example of this running in public repos: https://github.com/dmiller15/sync-test/pull/1

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

**Review by migbro — APPROVED** [review]
Gotta love a good ol' "test in prod"

File: .github/workflows/sync_pr_template.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
+on:
+  schedule:
+  - cron:  '57 0 1 * *'
```
Comment (yuankunzhu) [inline .github/workflows/sync_pr_template.yml:3]: the whole action setup looks ok to me in general, but this schedule looks like a monthly sync up, can we really test it in time? 

maybe you have already done this, but wondering if we really just wanna test this action, maybe we could try a shorter time interval, like a hourly or daily based schedule in one of our own repo or a private test repo?

**Review by yuankunzhu — COMMENTED** [review]
also wondering besides trigger this on a scheduled time, maybe we could also try trigger it by the bix-template repos merge event?

so it could be like, whenever the template repo has a PR merged, then check the doc of interest and do the PR here if there's changes.

more info around [Triggering GitHub Actions across different repositories](http://blog.marcnuri.com/triggering-github-actions-across-different-repositories/)

File: .github/workflows/sync_pr_template.yml (hunk 1 above)
Comment (dmiller15) [reply .github/workflows/sync_pr_template.yml:3]: Sure I can make it hourly for a day to see what happens.

**Comment by dmiller15** [comment]
@yuankunzhu Thanks for the info on repository dispatch. That seems like it could be useful but not in this context. It looks like the GHA in Repo A (template repo in our case) would need to know the names of every repo that is made from it so it's basically just a fancy controller repo. As a template repo, all the individual repos will inherit that template code, including the controller GHA code. So we're now asking people to edit the GHA that they get from the template. Like I said for @migbro suggestion, having a controller repo is a good idea and probably but not what I'm trying to accomplish here. I just want a template that everyone can use that will keep repos mostly up to date.

**Review by yuankunzhu — APPROVED** [review]

**Pull request merged**